dprint.json

```diff
@@ -0,0 +1,16 @@
+{
+  "excludes": [
+    // These files are auto-generated and don't need to be formatted
+    "./types/vscode"
+  ],
+  "plugins": [
+    "https://plugins.dprint.dev/typescript-0.85.1.wasm"
+  ],
+  "typescript": {
+    "arrowFunction.useParentheses": "preferNone",
```

Comment:
In the Prettier community, we have decided to set an equivalent of this option to `always` as this improves DX:
https://github.com/prettier/prettier/issues/6929

[Announcement (rationale)](https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-arrowparens-to-always-7430httpsgithubcomprettierprettierpull7430-by-kachkaevhttpsgithubcomkachkaev)

Reply:
Honestly, I wouldn't be upset if we switched to `always`. The original plan in #65993 was to not make changes to the original Prettier config. But if the DT folks feel strongly one way or another...

Reply:
I removed the overrides here, in #66235, and as an edit in the root discussion #65993. Thanks!